ogg123/ogg123.c

```diff
@@ -704,14 +707,19 @@ void play (const char *source_string)
       do {
 	
 	if (nthc-- == 0) {
-          if (audio_buffer) {
-            if (!buffer_submit_data(audio_buffer, convbuffer, ret)) {
+          int r;
+
+          if (audio_buffer)
+            r = buffer_submit_data(audio_buffer, convbuffer, ret);
+          else
+	    r = audio_play_callback(convbuffer, ret, eos, &audio_play_arg);
```

Comment:
This is the wrong indentation height, and have inconsistent use of space and tab.  Please adjust.